css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,34 @@
+/* This blacks out the whole things figure out how to avoid that
+tr:nth-child(odd) {
+  background-color: lightgray;
+}
+*/
+.odd {
+  background-color: lightgray;
+  color: black;
+}
+
+.thead {
+  background-color: black;
+  color: white;
+}
```

Comment:
Please try to complete this exercise without making any changes to the HTML document. The styling you have commented out above will be useful, you just may need to set some special styling on the `thead` or its contents in some way to override the general styling applied by `nth-child()`